content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -789,3 +805,23 @@ reclaimPolicy: Delete
 volumeBindingMode: WaitForFirstConsumer
 
 ```
+
+## Multi NAS Support
+The CSI PowerStore driver version 2.3.0 and later introduces multi-NAS support. This feature allows users to specify multiple NAS servers within a single storage class. By supporting multiple NAS servers in a single storage class, customers can create 2000 filesystems per PowerStore system. Previously, multiple storage classes were needed to support this configuration
+
+The following is a sample storage class file that supports multiple NAS servers for provisioning volumes:
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: powerstore-multi-nas
+provisioner: csi-powerstore.dell.com
+parameters:
+  nasServers: "nas-server-1,nas-server-2,nas-server-3"
+  storagePool: "pool-1"
```

Comment:
Do we support `storagePool` in PowerStore?

Reply:
Reverted